dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/pages/project/WorkFlowDefinitionPage.java

```diff
@@ -0,0 +1,205 @@
+        TaskDefinitionRequestData taskDefinitionRequestData = new TaskDefinitionRequestData();
+        TaskRelationRequestData taskRelationRequestData = new TaskRelationRequestData();
+
+        TaskParamsMap taskParams = new TaskParamsMap();
+        ArrayList<Object> localParams = new ArrayList<>();
+        ArrayList<Object> resourceList = new ArrayList<>();
+        taskParams.localParams(localParams);
+        taskParams.resourceList(resourceList);
+        taskParams.rawScript("echo 123");
+
+        taskDefinitionRequestData.code(genNumId);
+        taskDefinitionRequestData.delayTime("0");
+        taskDefinitionRequestData.description("");
+        taskDefinitionRequestData.environmentCode("-1");
+        taskDefinitionRequestData.failRetryInterval("1");
+        taskDefinitionRequestData.failRetryTimes("0");
+        taskDefinitionRequestData.flag("YES");
+        taskDefinitionRequestData.name("echo_123");
+        taskDefinitionRequestData.taskParams(taskParams);
+        taskDefinitionRequestData.taskPriority("MEDIUM");
+        taskDefinitionRequestData.taskType("SHELL");
+        taskDefinitionRequestData.timeout(0);
+        taskDefinitionRequestData.timeoutFlag("CLOSE");
+        taskDefinitionRequestData.timeoutNotifyStrategy("");
+        taskDefinitionRequestData.workerGroup("default");
+
+        ArrayList<Object> taskDefinitionRequestDataList = new ArrayList<>();
+        taskDefinitionRequestDataList.add(taskDefinitionRequestData);
+
+        HashMap<String, Object> conditionParams = new HashMap();
+
+        taskRelationRequestData.name("");
+        taskRelationRequestData.preTaskCode(0);
+        taskRelationRequestData.preTaskVersion(0);
+        taskRelationRequestData.preTaskCode(0);
+        taskRelationRequestData.postTaskCode(genNumId);
+        taskRelationRequestData.conditionType("NONE");
+        taskRelationRequestData.conditionParams(conditionParams);
+        ArrayList<Object> taskRelationRequestDataList = new ArrayList<>();
+        taskRelationRequestDataList.add(taskRelationRequestData);
+
+        HashMap<String, Object> locations = new HashMap();
+        locations.put("taskCode", genNumId);
+        locations.put("x", 33.5);
+        locations.put("y", 38.5);
+        ArrayList<Object> locationsList = new ArrayList<>();
+        locationsList.add(locations);
+
+        workFlowCreateRequestData.locations(JSONUtils.toJsonString(locationsList));
+        workFlowCreateRequestData.taskDefinitionJson(JSONUtils.toJsonString(taskDefinitionRequestDataList));
+        workFlowCreateRequestData.taskRelationJson(JSONUtils.toJsonString(taskRelationRequestDataList));
+        workFlowCreateRequestData.name(workFlowName);
+        workFlowCreateRequestData.tenantCode(TenantAPITest.tenantName);
+        workFlowCreateRequestData.executionType("PARALLEL");
+        workFlowCreateRequestData.description("");
+        workFlowCreateRequestData.globalParams("[]");
+        workFlowCreateRequestData.timeout(0);
```

Comment:
Same as L75.

Reply:
Thanks for the reminder, it has been fixed